rasterio/rio/info.py

```diff
@@ -99,6 +122,48 @@ def info(ctx, input, aspect, indent, namespace, meta_member, verbose, bidx,
                         float(band.mean())))
                 elif meta_member == 'checksum':
                     click.echo(str(src.checksum(bidx)))
+
+                elif meta_member == 'gcps':
+                    gcps, crs = src.gcps
+
+                    if geojson_type == 'collection':
+                        features = [p.__geo_interface__ for p in gcps]
+                        if projection == 'geographic' or precision >= 0:
+                            prec = 7 if precision == -1 else precision
+                            dst_crs = 'epsg:4326' if projection == 'geographic' else crs
+                            for i, feat in enumerate(features):
+                                geom = transform_geom(crs, dst_crs,
+                                                      feat['geometry'],
+                                                      precision=prec)
+                                feat['geometry'] = geom
+                                features[i] = feat
+
+                        click.echo(json.dumps(
+                            {'type': 'FeatureCollection',
+                             'features': features}))
+
+                    elif sequence:
+                        for p in gcps:
+                            if use_rs:
+                                click.echo(u'\x1e', nl=False)
+
+                            if geojson_type == 'feature':
```

Comment:
As it's written here, specifying `--feature` won't give you geojson unless you also specify `--sequence` along with it.